### PR TITLE
Blob Chat Fix

### PR DESCRIPTION
**Pull request by ThatDan123**
I tried to add a custom channel for the blob but seems to not appear, no idea why or what ive missed. So atm they can only speak in OOC, which is fine as they've got nothing else to talk to for now.

-Blob can only talk on OOC until above fixed
-Blob no longer talks on ghost
-Blob can only hear local now
-Fixes blob seeing ghosts
-Fixes blob spawned via event not getting objectives

CL: Stops blob hearing all chat, speaking in ghost chat
CL: Blob can no longer see ghosts
CL: Rebalanced blob damage
CL: Blobs now have names

### CODE REVIEW

File: UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs [hunk 1]
```diff
@@ -38,13 +38,14 @@ private struct DestroyChatMessage
...
 	public Color warningColor;
+	public Color blobColor;
 	public Color defaultColor;
 
 
 	/// <summary>
 	/// This channels can't be heared as sound by other players (like binary or changeling hivemind)
 	/// </summary>
-	public const ChatChannel NonVerbalChannels = ChatChannel.Binary | ChatChannel.Ghost;
+	public const ChatChannel NonVerbalChannels = ChatChannel.Binary | ChatChannel.Ghost | ChatChannel.Blob;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs:48]: Codacy found an issue: [Change this constant to a 'static' read-only property.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6358537)

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/Core/Chat/ChatContext/ChatInputContext.cs  1
- UnityProject/Assets/Scripts/Player/PlayerScript.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6358537&bid=20640679)

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/Player/PlayerScript.cs  3
- UnityProject/Assets/Scripts/Core/Chat/ChatContext/ChatInputContext.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6358537&bid=20640679)

**Comment by ThatDan123** [comment]
Ready to be reviewed

~Actually one thing left to do.~

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/Player/PlayerScript.cs  3
- UnityProject/Assets/Scripts/Core/Chat/ChatContext/ChatInputContext.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6358537&bid=20640679)

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventSpawnBlob.cs  1
- UnityProject/Assets/Scripts/Player/PlayerScript.cs  3
- UnityProject/Assets/Scripts/Core/Chat/ChatContext/ChatInputContext.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6358537&bid=20640679)

**Pull request merged**